frontend/src/components/cluster/Overview.tsx

```diff
@@ -136,12 +165,14 @@ function EventsSection() {
               >
                 <Box>{makeStatusLabel(event)}</Box>
               </LightTooltip>,
+            sort: false
```

Comment:
The default value is something I am not too confident giving a value for, but I am leaning towards it being `false`. i.e. if a caller wants to have sorting by default, they will need to just set `sort: true`.